app/Console/SendMail/RemindPost.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Console\SendMail;
+
+use App\Models\User;
+use App\Models\Post;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\RemindPost as Remind;
+
+class RemindPost
+{
+    public function __invoke()
+    {
+        $posts = Post::distinct()->select('user_id')->get();
```

Comment:
```suggestion
        $postUserIds = Post::distinct()->select('user_id')->get();
```

Mặc dù là query từ Post model nhưng vì đây là collection lưu thông tin user nên đổi tên lại cho dễ phân biệt.